### PR TITLE
Fast Pair features: software maturity

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -293,6 +293,96 @@ The following table indicates the software maturity levels of the support for Go
...
+.. toggle::
+
+   .. _software_maturity_fast_pair_feature:
+
+   .. list-table:: Google Fast Pair feature support
+      :header-rows: 1
+      :align: center
+      :widths: auto
+
+      * -
```
Comment (MarekPieta) [inline doc/nrf/releases_and_maturity/software_maturity.rst:307]: `Feature`?

**Review by MarekPieta — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst (hunk 1 above)
Comment (kapi-no) [reply doc/nrf/releases_and_maturity/software_maturity.rst:307]: I didn't add it so it looks the same as other feature tables. Should I add it?

Please take a look at the Protocol and Security feature tables.

Comment (MarekPieta) [reply doc/nrf/releases_and_maturity/software_maturity.rst:307]: We can leave it as is then

Comment (kapi-no) [reply doc/nrf/releases_and_maturity/software_maturity.rst:307]: I can add it if others agree that it is a step in the right direction

File: samples/bluetooth/fast_pair/input_device/README.rst [hunk 2]
```diff
@@ -14,8 +14,7 @@ Google also provides additional features built upon the Fast Pair standard.
 For detailed information about supported functionalities, see the official `Fast Pair`_ documentation.
 
 .. note::
-   The Fast Pair support in the |NCS| is :ref:`experimental <software_maturity>`.
-   See :ref:`ug_bt_fast_pair` for details.
+   The software maturity level for the input device use case is demonstrated in the :ref:`software_maturity_fast_pair_use_case` table.
```
Comment (divipillai) [inline samples/bluetooth/fast_pair/input_device/README.rst:17]:
```suggestion
   The software maturity level for the input device use case is listed in the :ref:`software_maturity_fast_pair_use_case` table.
```

File: samples/bluetooth/fast_pair/locator_tag/README.rst [hunk 3]
```diff
@@ -15,8 +15,7 @@ Google Fast Pair standard also supports the Find My Device Network (FMDN) extens
 For detailed information, see the official `Fast Pair Find My Device Network extension`_ documentation.
 
 .. note::
-   The Fast Pair support in the |NCS| is :ref:`experimental <software_maturity>`.
-   See :ref:`ug_bt_fast_pair` for details.
+   The software maturity level for the locator tag use case is demonstrated in the :ref:`software_maturity_fast_pair_use_case` table.
```
Comment (divipillai) [inline samples/bluetooth/fast_pair/locator_tag/README.rst:18]:
```suggestion
   The software maturity level for the locator tag use case is listed in the :ref:`software_maturity_fast_pair_use_case` table.
```

File: doc/nrf/releases_and_maturity/software_maturity.rst (hunk 1 above)
Comment (divipillai) [reply doc/nrf/releases_and_maturity/software_maturity.rst:307]: yeah, we could leave it.

**Review by divipillai — APPROVED** [review]

**Comment by kapi-no** [comment]
rebasing due to the stuck CI. No file diff has changed in the scope of this PR

**Comment by kapi-no** [comment]
@anangl, this is ready to be merged

**Pull request merged**